typescript/graphql-nestjs/src/prisma.service.ts

```diff
@@ -6,10 +6,4 @@ export class PrismaService extends PrismaClient implements OnModuleInit {
   async onModuleInit() {
     await this.$connect()
```

Comment:
```suggestion
    // Note: this is optional
    await this.$connect()
```
While you're doing this, I think we could add this comment